src/ansi-c/Makefile

```diff
@@ -121,16 +123,19 @@ ansi_c_internal_additions$(OBJEXT): arm_builtin_headers.inc
 ansi_c_internal_additions$(OBJEXT): cw_builtin_headers.inc
 ansi_c_internal_additions$(OBJEXT): clang_builtin_headers.inc
 
-generated_files: cprover_library.inc gcc_builtin_headers_generic.inc \
+generated_files: cprover_library.inc \
+	gcc_builtin_headers_types.inc \
+	gcc_builtin_headers_generic.inc \
 	gcc_builtin_headers_ia32.inc gcc_builtin_headers_alpha.inc \
 	gcc_builtin_headers_arm.inc gcc_builtin_headers_mips.inc \
 	gcc_builtin_headers_power.inc arm_builtin_headers.inc \
+	gcc_builtin_headers_ia32-2.inc \
+	gcc_builtin_headers_math.inc gcc_builtin_headers_omp.inc \
+	gcc_builtin_headers_tm.inc gcc_builtin_headers_ubsan.inc \
+	gcc_builtin_headers_mem_string.inc \
+	gcc_builtin_headers_ia32-3.inc gcc_builtin_headers_ia32-4.inc \
```

Comment:
Why is this order of file names better than the previous one?

Reply:
Now sorted